ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -609,6 +618,11 @@ ORK_CLASS_AVAILABLE
  */
 @property (readonly, getter=isVertical) BOOL vertical;
 
+/**
+ A Boolean value indicating whether the selected value should be hidden.
+ */
+@property (assign, getter=shouldHideSelectedValueLabel,setter=setHideSelectedValue:) BOOL hideSelectedValue;
+
 /**
  The colors to use when drawing a color gradient above the slider. Colors are drawn such that
```

Comment:
Also, do you think it is worth explicitly setting _hideSelectedValue to NO as part of the initializers?

Reply:
Yes, given the architectural differences per platform for BOOL. Will update momentarily.